docs/python_api/solvers/mesolve.md

```diff
@@ -1,3 +1,3 @@
-# Master equation
+# Lindblad Master Equation
```

Comment:
For consistency, we should have either lower case initials all around (as we do with "Quantum utilities" or "Tensor conversion" e.g.) or upper case. I personnaly prefer lower case because it's easier to read. But both are fine.

In any case, can you make the changes throughout the doc section titles?